Sources/Widgets/Representations/SplineContextRepresentation/index.js

```diff
@@ -65,6 +65,7 @@ function vtkSplineContextRepresentation(publicAPI, model) {
 
     const polydata = vtkPolyData.newInstance();
     const widgetState = inData[0];
+    model.close = widgetState.getIsSplineClosed();
```

Comment:
why do you want to keep model.close ?

Reply:
I suggest to remove `close` from `model` everywhere in this file, simply rely on `widgetState.getIsSplineClosed()`